web/views.py

```
@@ -7018,3 +7024,577 @@
+        end_time__lt=timezone.now()
+    ).order_by('-start_time')
+    
+    # Get chapter members with leadership roles first
+    members = chapter.members.filter(is_approved=True).order_by(
+        models.Case(
+            models.When(role='lead', then=0),
+            models.When(role='co_organizer', then=1),
+            models.When(role='volunteer', then=2),
+            default=3,
+        ),
+        'joined_at'
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)